Changes to formatting and clarity in settings.md (#32)
Kruptein/planarally-docs: docs/dm/settings.md

In fake player mode, you'll also be stuck on 1.0.

### Vision Mode

_This is a technical option that I intend to remove in the next release of PlanarAlly._

Historically, PlanarAlly calculated vision with bounding volume hierarchies, or bvh for short.
Later, a different method was introduced based on triangulation. 
This was initially an optional, experimental mode you could select using this vision mode setting.

As time went on, however, the triangle mode became the default, as it is much more potent.  
The option to choose the older mode remains for legacy reasons. 
It’s not being maintained currently.

### Minimal/Maximal vision

These settings are more tuning factors to the vision system. 
They determine limits to the vision of all tokens.

Suppose there is one giant, open field that stretches 5000 ft. and is completely illuminated.
One could expect the entire area to be visible to your token standing in the middle of this field.
There are, however, limitations to the human and nonhuman eye, hence the introduction of these settings.

Everything up to the minimal vision range can be seen as usual. 
Everything past the maximal range cannot be seen and is fog of war.  

> Everything in between is shown in a diminishing opacity as you go further towards the edge of vision.

Reviewer: ```suggestion
Everything in between is shown in an increasing opacity as you go further towards the edge of vision.
```

If I'm not mistaken (I may be, though), opacity is "not see-through-able" / the opposite of transparency, and, thus, towards the edge of the vision, opacity increases.

One could, of course, also show `in a diminishing transparency`.
